### ISSUE TITLE
just released talkback broke access to edit fields in the twitter app.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#156]
```
What steps will reproduce the problem?
1. If you have twitter, open it.
2. attempt to compose a new tweet.
3. Talkback speaks enter text here.

What is the expected output? What do you see instead?
Talkback should be speaking the text in the edit box, not "Enter text here."

What version of the product are you using? On what operating system?
3.2.1 Android 4.2

Please provide any additional information below.
In the previous version of Talkback, it spoke the text as it should.


```

Original issue reported on code.google.com by `jmwalto...@gmail.com` on 29 Nov 2012 at 2:09

**Comment by GoogleCodeExporter** [comment]
```
TalkBack is doing the right thing in this case.  The problem appears to be with 
the Twitter app itself.

The application is setting a content description on the 
MultiAutoCompleteTextView used in the tweet composition UI.  Whenever a 
developer sets a content description on a text view, they're explicitly 
overriding the behavior for that view.

Twitter should probably use the android:hint attribute instead to label this 
field.  This will cause TalkBack to speak "Enter tweet here" when there is no 
text in that field, and speak the contents of the field when text is present.

Information for developers on this topic can be found at: 
http://developer.android.com/guide/topics/ui/accessibility/apps.html#label-ui
```

Original comment by `caseybur...@google.com` on 4 Dec 2012 at 1:48
- Changed state: **Invalid**
